.github/workflows/check_schema_version.sh

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env bash
+
+# Collect error codes so we can see all the invalid json files in one run
+exitCode=0
+
+# $1 - takes a single argument of the path to the JSON file containing a schemaVersion key at the top level.
+function get_schema_version() {
+  node << EOF
+    const { readFileSync } = require('fs')
+    console.log(JSON.parse(readFileSync("./$1")).schemaVersion)
+EOF
+}
+
+function get_all_schemaVersion_defining_files () {
+  # look for all json files with 'schemaVersion": "[1-9]'
+  grep --include='*.json' --files-with-matches --recursive --word-regexp --regexp='schemaVersion": "[1-9]' source | \
```

Comment:
`ajv` operates on YAML files, which is what we do in [source/unified-test-format/tests/Makefile](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/tests/Makefile). Can this be changed to target those instead?

I suppose it's not a big issue if that change isn't feasible, since [json-regenerate-check.yml](https://github.com/mongodb/specifications/blob/master/.github/workflows/json-regenerate-check.yml) should already enforce that YAML and JSON are in sync with one another.

Reply:
Done, I had to add js-yaml to the deps install step, and modified the grep regex to account for the mixed quoting.